source/contracts/personal_edition.md

```diff
@@ -0,0 +1,93 @@
+# Safe Personal Edition Smart Contract
+
+This version is targeted at users that control all keys owning a safe. The transaction hash can be signed with the private keys that manage the safe. 
+
+Once the required number of confirmations is available `execTransactionAndPaySubmitter` can be called with the sending confirmation signatures. This method will pay the submitter of the transaction for the transaction fees after the Safe transaction has been executed.
+
+## Safe Contract Deployment
+The Gnosis Safe Personal Edition smart contract was written with the usage of a proxy contract in mind. Because of that there is no constructor and it is required to call an inilize function on the contract before it can be used. For this it is recommended to use the ProxyFactory or the DelegatingConstructorProxy.
+
+Using the ProxyFactory or deploying a proxy requires that the user has ether on an externally owned account. To make it possible to pay for the creation with any token or ether the following flow is used.
+
+1. Create deployment transaction. The PayingProxy enables the payment in any ERC20 token. Once the proxy is deployed it will refund a predefined address with the funds present at the address where it was deployed.
+1. To make the deployed address deterministig it is necessary to use a known account and calculate the target address. To make this trustless it is recommended to use a random account that has nonce 0. This can be done by creating a random signature for the deployment transaction. From that transaction it is possible to derive the sender and the target address.
+1. The user needs to transfer at least the amount required for the payment to the target address.
+1. Once the payment is present at the target address the relay service will fund the sender with ether required for the creation transaction.
+1. As soon as the sender is funded the creation transaction can be submmited.
+
```

Comment:
The steps are correct, the video is cool, but I would still add a diagram here. This part is, based on my opinion, one of the best things we have done and, in some way, a point to exploit some "marketing", so invest a bit of more time giving further explanations would be good.

But, again, it is just my 2 cents.

Reply:
Will do that later, maybe also a separate section: https://github.com/gnosis/safe-docs/issues/2